config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/environment/EnvironmentBundleUtils.java

```diff
@@ -57,8 +57,13 @@ public static Bundle getDeploymentBundle() {
     }
 
     static void processDeploymentBundles(Bundle environmentBundle,
-                                          List<TemplatizedBundle> templatizedBundles,
-                                          EnvironmentBundleCreationMode mode) {
+                                         List<TemplatizedBundle> templatizedBundles,
+                                         EnvironmentBundleCreationMode mode,
+                                         boolean detemplatize) {
+        if (!detemplatize) {
+            return;
```

Comment:
Should we add a log here?

Reply:
We can, but this boolean is specified by the user, so if they set this it's kind of clear that this will happen :)
Otherwise, behaviour is the same as previous.

Reply:
sounds good to me.